lib/bot.js

```diff
@@ -335,6 +347,44 @@ class Bot {
     return null;
   }
 
+  findWebhook(ircChannel) {
+    const discordChannelName = this.invertedMapping[ircChannel.toLowerCase()];
+    if (discordChannelName) {
+      const webhook = this.webhooks[discordChannelName];
+      if (webhook) {
+        return webhook;
+      }
+    }
+    return null;
+  }
+
+  getDiscordAvatar(nick, channel) {
+    const guildMembers = this.findDiscordChannel(channel).guild.members;
+    const findByNicknameOrUsername = caseSensitive =>
+      (member) => {
+        if (caseSensitive) {
```

Comment:
You can even have two users with the exact same nickname: http://i.imgur.com/ACuYlcS.png + http://i.imgur.com/oD3JEfK.png
(And @maorninja suggested the change :) )